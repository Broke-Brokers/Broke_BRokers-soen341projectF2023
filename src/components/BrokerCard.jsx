import React from 'react';
<<<<<<< HEAD:src/components/property-page/BrokerCard.jsx
import '../src/components/property-page/BrokerCard.css' 
=======
import '../components/BrokerCard.css' 
>>>>>>> parent of 8c30fae (created a property-page folder and broke everything.):src/components/BrokerCard.jsx


export default function BrokerCard(props){

/*const brokerName = props.name;
   const email = props.email;
   const phoneNumber = props.phoneNumber;
    */

   /*For testing purposes these are hardcoded, please remove after testing*/
   const brokerName = "Albus Dumbledore";
   const email = "itsleviosa@hogwarts.com";
   const phoneNumber = "514-252-2525";
    const profilePicture = "https://pointrussell.opencities.com/files/content/public/v/5/council/elected-members/albus-dumbledore/albus-dumbledore.jpg?dimension=pageimage&w=480";


    return(
        <>
        <div className="brokerCard">
       <div className="cardTitle">Property Broker</div>
  <div className="profilePic">
  
    <img src={profilePicture} alt={brokerName} />
  </div>
  <div className="brokerName">
    <h2>{brokerName}</h2>
  </div>
  <div className="brokerInfo">
   <p>{email}</p>
   <p>{phoneNumber}</p>
  </div>
  <div className="social">
    <ul>
      <li><i class="fab fa-facebook"></i></li>
      <li><i class="fab fa-twitter"></i></li>
      <li><i class="fab fa-github"></i></li>
      <li><i class="fab fa-dev"></i></li>
      <li><i class="fas fa-link"></i></li>
    </ul>
  </div>
</div>
</>
        
        );
        }
        