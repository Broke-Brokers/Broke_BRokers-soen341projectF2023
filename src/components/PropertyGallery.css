
.myGalleryCountainer{
width: 50%;
float: right;

}

