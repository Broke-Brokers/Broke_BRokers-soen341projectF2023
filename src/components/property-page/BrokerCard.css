
@media screen and (min-width: 600px) {
  .brokerCard {
    max-width:100%;
  }
}

.brokerCard {
  border-left:  30px solid #EE7EA0;
  border-top : 1px solid #d2d2d2;
  box-shadow: 5px 6px 0px #D2D2D2;
  transition: 0.3s all;
  border-radius: 3px;
  background-color: #fff;
  padding: 21px;
  display: flex;
  flex-direction: column;
  justify-content: center;
  text-align: center;
  /*min-width: 340px;*/
 max-width: 30%;
 display: inline-block;
 margin:1%;
 vertical-align: center;

  .profilePic {
    img {
      border-radius: 50%;
      border: 7px solid #EE7EA0;
      max-height:50%;
      max-width: 50%;
    
    }
  }

  .brokerName {
    font-size: 26px;
    margin-top: 18px;
    color: #EE7EA0;
  }

  .brokerInfo {
    font-size: 17px;
    margin-top: 18px;
    color: #000;
    word-wrap: break-word;
  }
  .cardTitle {
    font-size: 26px;
    margin-top: 18px;
    color: #D2D2D2;
    padding:5%;
    word-wrap: break-word;
  }
  .social {
    ul {
      margin-top: 22px;
      list-style-type: none;
      li {
        display: inline;
        font-size: 22px;
        cursor: pointer;
        color: #D2D2D2;
        padding: 1%;
      }
    }
  }
}
