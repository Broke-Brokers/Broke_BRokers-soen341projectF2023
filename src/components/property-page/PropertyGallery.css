
.myGalleryContainer{
width: 100%;
/*float: right;*/
/*border-style: solid;
border-width: 7px;*/
margin :0px;
padding :0px;
}

