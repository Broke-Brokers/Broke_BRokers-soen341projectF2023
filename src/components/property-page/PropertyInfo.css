
.infoContainer{
    border-left:  30px solid #E1CFCA;
    border-top : 1px solid #d2d2d2;
    box-shadow: 5px 6px 0px #D2D2D2;
    transition: 0.3s all;
    border-radius: 3px;
    background-color: #fff;
    padding: 21px;
    display: inline-block;
    justify-content: center;
    text-align: center;
    min-width: 340px;
    width: 60%;
    max-width: 95%;
    margin:1%;

}



.propertyInfoBox{
    width: 100%;
   padding: 20px;
   /*margin: 10px;*/
  /* clear: both;*/
   display: inline-block;
 vertical-align: top;
 text-align:left;
 
}


h1{
   
    color: #EE7EA0;
   font-family : 'PT Sans', sans-serif;
   font-size:48px;
   font-weight: 700;
   text-align : left;
}

.photoContainer{
    width: 100%;
    float: left;
    padding: 20px;
    }

.photoContainer img{
        max-width: 100%;
        height: auto;
        }
i{
    color: #EE7EA0;
}


/* Offer Button style */
.offer-button {
    border-radius: 5px;
    background-color: rgb(177, 95, 95);
    color: white;
    padding: 20px 30px;
    text-decoration: none;
    border: none;
    cursor: pointer;
    float:left;
    margin : 5%;
  }
  
  /* Button hover effect (optional) */
  .offer-button:hover {
    background-color: rgb(206, 21, 21);

  }