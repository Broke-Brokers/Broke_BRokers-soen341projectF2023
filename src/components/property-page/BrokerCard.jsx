import React, {useRef}from 'react';
import '../property-page/BrokerCard.css' 
import Admin_Brokers_Grid from '../Profile-page/Admin_Brokers_Grid';

//FOR CRUD BROKER AS AN ADMIN
// Broker card is parent component
// Child component is admin_broker_grid
// function in admin broker grid is called in broker card
// useRef: reference of child component to access it









export default function BrokerCard({ brokerList }){
  let brokerName, email, phoneNumber; // Declare these variables in the parent scope

  if (!brokerList) { // evaluates to true if brokerList is null
     //Placeholder name to handle the case where brokerList is empty
     brokerName = "Albus Dumbledore";
     email = "itsleviosa@hogwarts.com";
     phoneNumber = "514-252-2525";
  }
  else{
  
    brokerName = brokerList.FirstName;
    email = brokerList.Email;
    phoneNumber = brokerList.PhoneNumber;

   }
   /*The fields:
     Broker_id integer PK
  First_Name VARCHAR
  Last_Name VARCHAR
  Phone_Number VARCHAR
  Email Varchar
   */
    

  


    const profilePicture = "https://pointrussell.opencities.com/files/content/public/v/5/council/elected-members/albus-dumbledore/albus-dumbledore.jpg?dimension=pageimage&w=480";
    const RefChild_brokerGrid = useRef();
    <Admin_Brokers_Grid ref_BrokerGrid={RefChild_brokerGrid}/>


    return(
        <>
        <div className="brokerCard">
       <div className="cardTitle">Property Broker</div>
  <div className="profilePic">
  
    <img src={profilePicture} alt={brokerName} />
  </div>
  <div className="brokerName">
    <h2>{brokerName}</h2>
  </div>
  <div className="brokerInfo">
   <p>{email}</p>
   <p>{phoneNumber}</p>
  </div>
  <div className="social">
    <ul>
      <li><i class="fab fa-facebook"></i></li>
      <li><i class="fas fa-envelope"></i></li>
            <li><i class="fas fa-phone"></i></li>
      <li><i class="fas fa-link"></i></li>
    </ul>
  </div>
 

 
  <button onClick={()=> RefChild_brokerGrid.current.modifyBroker(brokerName,email,phoneNumber)}>Modify</button>
  

  
  
  
  
  
</div>

</>
        
        );
        }
        