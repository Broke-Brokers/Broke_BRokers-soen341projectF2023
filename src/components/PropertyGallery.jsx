import React from 'react';
import ImageGallery from "react-image-gallery";
<<<<<<< HEAD:src/components/property-page/PropertyGallery.jsx
import '../src/components/property-page/PropertyGallery.css' 
=======
import '../components/PropertyGallery.css' 
>>>>>>> parent of 8c30fae (created a property-page folder and broke everything.):src/components/PropertyGallery.jsx
import 'react-image-gallery/styles/css/image-gallery.css';

export default function PropertyGallery(){

const images = [
  {
    original: "https://picsum.photos/id/1018/1000/600/",
    thumbnail: "https://picsum.photos/id/1018/250/150/",
  },
  {
    original: "https://picsum.photos/id/1015/1000/600/",
    thumbnail: "https://picsum.photos/id/1015/250/150/",
  },
  {
    original: "https://picsum.photos/id/1019/1000/600/",
    thumbnail: "https://picsum.photos/id/1019/250/150/",
  },
];


return(
<div className="myGalleryContainer">
    
<ImageGallery
thumbnailPosition={'right'}
showPlayButton={false}
items={images} />
    </div>

);
}

/*https://www.npmjs.com/package/react-image-gallery*/