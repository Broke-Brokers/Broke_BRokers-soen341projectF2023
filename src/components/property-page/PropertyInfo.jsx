import React from 'react'
import'../property-page/PropertyInfo.css';
import home_photo from '../../Images/property_image_test1.jpg'
import SaveFavorite_Button from '../buttons/SaveFavorite_Button';
import RequestVisit_Button from '../buttons/RequestVisit_Button';
import Popup from 'reactjs-popup';
import 'reactjs-popup/dist/index.css';
import Request_visit2 from '../forms/Request_visit2';
import { Link } from 'react-router-dom';

export default function PropertyInfo(props){


   /* The fields:
     Price integer
  Adress Varchar
  City Varchar
  Province Varchar
  Neghborhood Varchar
  No_bathrooms integer
  No_Bedrooms integer
   */     

   /*const price = props.price;
   const postName = props.postName;
   const constructionYear = props.constrcutionYear;
   const price =  props.price;
    const Adress = props.Adress;
    const City= props.City;
    const Province = props.Province;
    const Neghborhood= props.Neghborhood;
    const No_bathrooms= props.No_bathroom;
    const No_Bedrooms = props.No_Bedrooms;
   const postName = "postName";
   const constructionYear = 1966;
    */

   /*For testing purposes these are hardcoded, please remove after testing*/
    const price = 12345;
    const Adress = 1966;
    const City= "Montreal";
    const Province = "Quebec";
    const Neghborhood="Centre-Ville";
    const No_bathrooms= 2;
    const No_Bedrooms = 5;
   const postName = "postName";
   const constructionYear = 1966;
 
    return(

        <div className="infoContainer">

            {/**Css of photoContainer and photochontainer will need to be moved to a photo component. */}
      <div className="photoContainer">
      <img src={home_photo} alt=''/>
      </div>
    <div className="propertyInfoBox">
        <h1>{price}$</h1>
        <h2>{postName}</h2>
        <p>{Adress}, {City}, {Province}</p>
        <p>{Neghborhood}</p>
        
        <p><i class="fa fa-bed"></i> Number of bedrooms: {No_Bedrooms}</p>
       
        <p><i class="fa fa-bath"></i> Number of bathrooms: {No_bathrooms}</p>
      
        </div>
        <SaveFavorite_Button/>
        <Link to='/requestVisitForm'>
        <RequestVisit_Button/>
        </Link>
        
       <div >

        {/*We would need to style the pop up if we use this method https://react-popup.elazizi.com/css-styling/ */}
        <Popup trigger={<img className="requestButton" src={require('../../Images/home_visit.png')} alt="requestAVisit" /*onClick={() => requestAVistit()}*/ />} position="right center">
    <div><Request_visit2 /></div>
          </Popup>
    

          
        </div>
    </div>
    );
    }
    
    /*const requestAVistit= () => {
      /*<Popup trigger={<button> Trigger</button>} position="right center">
    <div>Popup content here 1!!</div>
  </Popup>
    }*/

    