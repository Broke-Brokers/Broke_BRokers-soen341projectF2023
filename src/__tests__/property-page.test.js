import React from 'react';
import renderer from 'react-test-renderer';

import BrokerCard from '../components/property-page/BrokerCard';
import PropertyGallery from '../components/property-page/PropertyGallery';
import PropertyInfo from '../components/property-page/PropertyInfo';


it('performs snapshot testing', () => {
const tree = renderer.create(<BrokerCard />).toJSON();
expect(tree).toMatchSnapshot();
});


it('performs snapshot testing', () => {
    const tree = renderer.create(<PropertyGallery />).toJSON();
    expect(tree).toMatchSnapshot();
    });

