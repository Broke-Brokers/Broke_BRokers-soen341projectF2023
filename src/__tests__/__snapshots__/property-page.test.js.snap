// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`performs snapshot testing 1`] = `
<div
  className="brokerCard"
>
  <div
    className="cardTitle"
  >
    Property Broker
  </div>
  <div
    className="profilePic"
  >
    <img
      alt="Albus Dumbledore"
      src="https://pointrussell.opencities.com/files/content/public/v/5/council/elected-members/albus-dumbledore/albus-dumbledore.jpg?dimension=pageimage&w=480"
    />
  </div>
  <div
    className="brokerName"
  >
    <h2>
      Albus Dumbledore
    </h2>
  </div>
  <div
    className="brokerInfo"
  >
    <p>
      itsleviosa@hogwarts.com
    </p>
    <p>
      514-252-2525
    </p>
  </div>
  <div
    className="social"
  >
    <ul>
      <li>
        <i
          class="fab fa-facebook"
        />
      </li>
      <li>
        <i
          class="fas fa-envelope"
        />
      </li>
      <li>
        <i
          class="fas fa-phone"
        />
      </li>
      <li>
        <i
          class="fas fa-link"
        />
      </li>
    </ul>
  </div>
  <button
    onClick={[Function]}
  >
    Modify
  </button>
</div>
`;

exports[`performs snapshot testing 2`] = `
<div
  className="myGalleryContainer"
>
  <div
    aria-live="polite"
    className="image-gallery"
  >
    <div
      className="image-gallery-content  image-gallery-thumbnails-right"
    >
      <div
        className="image-gallery-slide-wrapper  image-gallery-thumbnails-right"
      >
        <button
          aria-label="Previous Slide"
          className="image-gallery-icon image-gallery-left-nav"
          disabled={false}
          onClick={[Function]}
          type="button"
        >
          <svg
            className="image-gallery-svg"
            fill="none"
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
            strokeWidth={1}
            viewBox="6 0 12 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <polyline
              points="15 18 9 12 15 6"
            />
          </svg>
        </button>
        <button
          aria-label="Next Slide"
          className="image-gallery-icon image-gallery-right-nav"
          disabled={false}
          onClick={[Function]}
          type="button"
        >
          <svg
            className="image-gallery-svg"
            fill="none"
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
            strokeWidth={1}
            viewBox="6 0 12 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <polyline
              points="9 18 15 12 9 6"
            />
          </svg>
        </button>
        <div
          className="image-gallery-swipe"
        >
          <div
            className="image-gallery-slides"
          >
            <div
              aria-label="Go to Slide 1"
              className="image-gallery-slide  image-gallery-center "
              onClick={null}
              onFocus={null}
              onKeyUp={[Function]}
              onMouseLeave={null}
              onMouseOver={null}
              onTouchEnd={null}
              onTouchMove={null}
              onTouchStart={null}
              role="button"
              style={
                Object {
                  "MozTransform": "translate3d(0%, 0, 0)",
                  "OTransform": "translate3d(0%, 0, 0)",
                  "WebkitTransform": "translate3d(0%, 0, 0)",
                  "display": "inherit",
                  "msTransform": "translate3d(0%, 0, 0)",
                  "transform": "translate3d(0%, 0, 0)",
                }
              }
              tabIndex="-1"
            >
              <img
                className="image-gallery-image"
                onError={[Function]}
                onLoad={[Function]}
                src="https://picsum.photos/id/1018/1000/600/"
              />
            </div>
            <div
              aria-label="Go to Slide 2"
              className="image-gallery-slide  image-gallery-right "
              onClick={null}
              onFocus={null}
              onKeyUp={[Function]}
              onMouseLeave={null}
              onMouseOver={null}
              onTouchEnd={null}
              onTouchMove={null}
              onTouchStart={null}
              role="button"
              style={
                Object {
                  "MozTransform": "translate3d(100%, 0, 0)",
                  "OTransform": "translate3d(100%, 0, 0)",
                  "WebkitTransform": "translate3d(100%, 0, 0)",
                  "display": "inherit",
                  "msTransform": "translate3d(100%, 0, 0)",
                  "transform": "translate3d(100%, 0, 0)",
                }
              }
              tabIndex="-1"
            >
              <img
                className="image-gallery-image"
                onError={[Function]}
                onLoad={[Function]}
                src="https://picsum.photos/id/1015/1000/600/"
              />
            </div>
            <div
              aria-label="Go to Slide 3"
              className="image-gallery-slide  "
              onClick={null}
              onFocus={null}
              onKeyUp={[Function]}
              onMouseLeave={null}
              onMouseOver={null}
              onTouchEnd={null}
              onTouchMove={null}
              onTouchStart={null}
              role="button"
              style={
                Object {
                  "MozTransform": "translate3d(200%, 0, 0)",
                  "OTransform": "translate3d(200%, 0, 0)",
                  "WebkitTransform": "translate3d(200%, 0, 0)",
                  "display": "inherit",
                  "msTransform": "translate3d(200%, 0, 0)",
                  "transform": "translate3d(200%, 0, 0)",
                }
              }
              tabIndex="-1"
            >
              <img
                className="image-gallery-image"
                onError={[Function]}
                onLoad={[Function]}
                src="https://picsum.photos/id/1019/1000/600/"
              />
            </div>
            <div
              aria-label="Go to Slide 4"
              className="image-gallery-slide  "
              onClick={null}
              onFocus={null}
              onKeyUp={[Function]}
              onMouseLeave={null}
              onMouseOver={null}
              onTouchEnd={null}
              onTouchMove={null}
              onTouchStart={null}
              role="button"
              style={
                Object {
                  "MozTransform": "translate3d(300%, 0, 0)",
                  "OTransform": "translate3d(300%, 0, 0)",
                  "WebkitTransform": "translate3d(300%, 0, 0)",
                  "display": "inherit",
                  "msTransform": "translate3d(300%, 0, 0)",
                  "transform": "translate3d(300%, 0, 0)",
                }
              }
              tabIndex="-1"
            >
              <img
                className="image-gallery-image"
                onError={[Function]}
                onLoad={[Function]}
                src="https://picsum.photos/id/1019/1000/600/"
              />
            </div>
            <div
              aria-label="Go to Slide 5"
              className="image-gallery-slide  "
              onClick={null}
              onFocus={null}
              onKeyUp={[Function]}
              onMouseLeave={null}
              onMouseOver={null}
              onTouchEnd={null}
              onTouchMove={null}
              onTouchStart={null}
              role="button"
              style={
                Object {
                  "MozTransform": "translate3d(400%, 0, 0)",
                  "OTransform": "translate3d(400%, 0, 0)",
                  "WebkitTransform": "translate3d(400%, 0, 0)",
                  "display": "inherit",
                  "msTransform": "translate3d(400%, 0, 0)",
                  "transform": "translate3d(400%, 0, 0)",
                }
              }
              tabIndex="-1"
            >
              <img
                className="image-gallery-image"
                onError={[Function]}
                onLoad={[Function]}
                src="https://picsum.photos/id/1018/1000/600/"
              />
            </div>
            <div
              aria-label="Go to Slide 6"
              className="image-gallery-slide  "
              onClick={null}
              onFocus={null}
              onKeyUp={[Function]}
              onMouseLeave={null}
              onMouseOver={null}
              onTouchEnd={null}
              onTouchMove={null}
              onTouchStart={null}
              role="button"
              style={
                Object {
                  "MozTransform": "translate3d(500%, 0, 0)",
                  "OTransform": "translate3d(500%, 0, 0)",
                  "WebkitTransform": "translate3d(500%, 0, 0)",
                  "display": "inherit",
                  "msTransform": "translate3d(500%, 0, 0)",
                  "transform": "translate3d(500%, 0, 0)",
                }
              }
              tabIndex="-1"
            >
              <img
                className="image-gallery-image"
                onError={[Function]}
                onLoad={[Function]}
                src="https://picsum.photos/id/1019/1000/600/"
              />
            </div>
            <div
              aria-label="Go to Slide 7"
              className="image-gallery-slide  image-gallery-left "
              onClick={null}
              onFocus={null}
              onKeyUp={[Function]}
              onMouseLeave={null}
              onMouseOver={null}
              onTouchEnd={null}
              onTouchMove={null}
              onTouchStart={null}
              role="button"
              style={
                Object {
                  "MozTransform": "translate3d(-100%, 0, 0)",
                  "OTransform": "translate3d(-100%, 0, 0)",
                  "WebkitTransform": "translate3d(-100%, 0, 0)",
                  "display": "inherit",
                  "msTransform": "translate3d(-100%, 0, 0)",
                  "transform": "translate3d(-100%, 0, 0)",
                }
              }
              tabIndex="-1"
            >
              <img
                className="image-gallery-image"
                onError={[Function]}
                onLoad={[Function]}
                src="https://picsum.photos/id/1015/1000/600/"
              />
            </div>
          </div>
        </div>
        <button
          aria-label="Open Fullscreen"
          className="image-gallery-icon image-gallery-fullscreen-button"
          onClick={[Function]}
          type="button"
        >
          <svg
            className="image-gallery-svg"
            fill="none"
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
            strokeWidth={2}
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M8 3H5a2 2 0 0 0-2 2v3m18 0V5a2 2 0 0 0-2-2h-3m0 18h3a2 2 0 0 0 2-2v-3M3 16v3a2 2 0 0 0 2 2h3"
            />
          </svg>
        </button>
      </div>
      <div
        className="image-gallery-thumbnails-wrapper  image-gallery-thumbnails-right thumbnails-swipe-vertical"
      >
        <div
          className="image-gallery-thumbnails"
          style={
            Object {
              "height": undefined,
            }
          }
        >
          <nav
            aria-label="Thumbnail Navigation"
            className="image-gallery-thumbnails-container"
            style={
              Object {
                "MozTransform": "translate3d(0, 0px, 0)",
                "OTransform": "translate3d(0, 0px, 0)",
                "WebkitTransform": "translate3d(0, 0px, 0)",
                "msTransform": "translate3d(0, 0px, 0)",
                "transform": "translate3d(0, 0px, 0)",
                "transition": "all 450ms ease-out",
              }
            }
          >
            <button
              aria-label="Go to Slide 1"
              aria-pressed="true"
              className="image-gallery-thumbnail active"
              onClick={[Function]}
              onFocus={[Function]}
              onKeyUp={[Function]}
              onMouseLeave={null}
              onMouseOver={[Function]}
              tabIndex="0"
              type="button"
            >
              <span
                className="image-gallery-thumbnail-inner"
              >
                <img
                  className="image-gallery-thumbnail-image"
                  onError={[Function]}
                  src="https://picsum.photos/id/1018/250/150/"
                />
              </span>
            </button>
            <button
              aria-label="Go to Slide 2"
              aria-pressed="false"
              className="image-gallery-thumbnail"
              onClick={[Function]}
              onFocus={[Function]}
              onKeyUp={[Function]}
              onMouseLeave={null}
              onMouseOver={[Function]}
              tabIndex="0"
              type="button"
            >
              <span
                className="image-gallery-thumbnail-inner"
              >
                <img
                  className="image-gallery-thumbnail-image"
                  onError={[Function]}
                  src="https://picsum.photos/id/1015/250/150/"
                />
              </span>
            </button>
            <button
              aria-label="Go to Slide 3"
              aria-pressed="false"
              className="image-gallery-thumbnail"
              onClick={[Function]}
              onFocus={[Function]}
              onKeyUp={[Function]}
              onMouseLeave={null}
              onMouseOver={[Function]}
              tabIndex="0"
              type="button"
            >
              <span
                className="image-gallery-thumbnail-inner"
              >
                <img
                  className="image-gallery-thumbnail-image"
                  onError={[Function]}
                  src="https://picsum.photos/id/1019/250/150/"
                />
              </span>
            </button>
            <button
              aria-label="Go to Slide 4"
              aria-pressed="false"
              className="image-gallery-thumbnail"
              onClick={[Function]}
              onFocus={[Function]}
              onKeyUp={[Function]}
              onMouseLeave={null}
              onMouseOver={[Function]}
              tabIndex="0"
              type="button"
            >
              <span
                className="image-gallery-thumbnail-inner"
              >
                <img
                  className="image-gallery-thumbnail-image"
                  onError={[Function]}
                  src="https://picsum.photos/id/1019/250/150/"
                />
              </span>
            </button>
            <button
              aria-label="Go to Slide 5"
              aria-pressed="false"
              className="image-gallery-thumbnail"
              onClick={[Function]}
              onFocus={[Function]}
              onKeyUp={[Function]}
              onMouseLeave={null}
              onMouseOver={[Function]}
              tabIndex="0"
              type="button"
            >
              <span
                className="image-gallery-thumbnail-inner"
              >
                <img
                  className="image-gallery-thumbnail-image"
                  onError={[Function]}
                  src="https://picsum.photos/id/1018/250/150/"
                />
              </span>
            </button>
            <button
              aria-label="Go to Slide 6"
              aria-pressed="false"
              className="image-gallery-thumbnail"
              onClick={[Function]}
              onFocus={[Function]}
              onKeyUp={[Function]}
              onMouseLeave={null}
              onMouseOver={[Function]}
              tabIndex="0"
              type="button"
            >
              <span
                className="image-gallery-thumbnail-inner"
              >
                <img
                  className="image-gallery-thumbnail-image"
                  onError={[Function]}
                  src="https://picsum.photos/id/1019/250/150/"
                />
              </span>
            </button>
            <button
              aria-label="Go to Slide 7"
              aria-pressed="false"
              className="image-gallery-thumbnail"
              onClick={[Function]}
              onFocus={[Function]}
              onKeyUp={[Function]}
              onMouseLeave={null}
              onMouseOver={[Function]}
              tabIndex="0"
              type="button"
            >
              <span
                className="image-gallery-thumbnail-inner"
              >
                <img
                  className="image-gallery-thumbnail-image"
                  onError={[Function]}
                  src="https://picsum.photos/id/1015/250/150/"
                />
              </span>
            </button>
          </nav>
        </div>
      </div>
    </div>
  </div>
</div>
`;
